client/rpcserver/handlers.go

```diff
@@ -452,4 +469,26 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	loginRoute: {
+		pwArgsShort: `"appPass"`,
+		argsShort:   ``,
+		cmdSummary:  `Attempt to login to all known DEX servers.`,
+		pwArgsLong: `Password Args:
+    appPass (string): The dex client password.`,
+		argsLong: ``,
+		returns: `Returns:
+    obj: An array of notifications.
+    [
+      {
+        "type" (string): The notification type.
+        "subject" (string): A clarification of type.
+        "details"(string): The notification details.
+        "severity" (int): The importance of the notification on a scale of 0
+	  through 5.
+        "stamp" (int): Unix time of the notification. Seconds since 00:00:00 Jan 1 1970.
+        "acked" (bool): Whether the notification was acknowledged.
+        "id" (string): A unique hex ID.
+      },...,
```

Comment:
I'm thinking we can do more than just marshal the 10 most recent notifications that `(*Core).Login` pulls.  It's also very tricky given how `Login` and `initializeDEXConnections` operate.  But first there are other pieces of data that can be obtained other than just notifications.  One is the active trades (`loaded`), whether it's just a count of these or an array of the order ids (with targeted cancel ids for cancel orders), which would require another return from `(*Core).resolveActiveTrades`.  This also generates notifications in the event of errors from DB loading of trades and trade resumption, which by the way can also create notifications for swap errors (actually I just noticed that `resumeTrades` always returns `nil`, so that's odd).

Another piece of summary data might be the number of dex connections passed and failed authorization, with their urls/hosts and account IDs.  It's a little tricky that we want to provide results and any errors from `(*Core).initializeDEXConnections` because that function just notifies and does not return anything.  We may wish to modify it with an output, or write another `Core` method to check all the dexConnections with perhaps `(*dexAccount).authed` like

```go
type dexAcct struct{dex, account string}
var authed, unauthed []dexAcct
for _, dc := range c.conns {
	// Copy the iterator for use in the authDEX goroutine.
	if dc.acct.authed() {
		authed = append(authed, dexAcct{dc.acct.url, dc.acct.ID()}
...
```

But it might be best to know what error prevented a given dex from authing.  So perhaps `initializeDEXConnections` could return a `[]results`, with one `result` for each entry in `c.conns` at the time of calling where `type result struct{host string, acct account.AccountID, ..., err error}`?

I can see that the notifications would be helpful, but they will also be logged by `dexc`, so I'm not sure returning them is what we want.  I'd feel  better about it though if we knew which notifications are related to the Login call.  There could be more or less than 10...

Reply:
The 10 notifications are maybe not at home as a `Login` return anyways. If it's cleaner, we can move those to a new **Core** method.

Reply:
Maybe a `LoginResult` type that has fields for login-related data?